ls1intum/Artemis
# `General`: Add README to data export

# Needs to be tested on a test server with mail support, e.g. ts1



### Checklist
#### General


#### Server
### Motivation and Context
To make our data export compliant with Art. 15 GDPR we have to provide a README file that contains specific information as specified by Art. 15

### Description
Added the README file. The file is added to every data export 
### Steps for Testing

Prerequisites:
- 1 User with an email address you have access to

1. Log in to Artemis
2. Navigate to Privacy
3. Navigate to Data Export
4. Request a data export
5. Wait until you receive your email
6. Download the data export
7. Make sure the zip contains the README.md file


### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage






#### Server

| Class/File | Line Coverage | Confirmation (assert/expect) |
|------------|--------------:|---------------------:|
| [DataExportCreationService.java](https://bamboo.ase.in.tum.de/browse/ARTEMIS-TESTS5135/latest/artifact/shared/Coverage-Report-Server-Tests/de.tum.in.www1.artemis.service.dataexport/DataExportCreationService.html) | 96% | ✅ |
| [DataExportService.java](https://bamboo.ase.in.tum.de/browse/ARTEMIS-TESTS5135/latest/artifact/shared/Coverage-Report-Server-Tests/de.tum.in.www1.artemis.service.dataexport/DataExportService.html) | 99% | ✅|
| [DataExportResource.java](https://bamboo.ase.in.tum.de/browse/ARTEMIS-TESTS5135/latest/artifact/shared/Coverage-Report-Server-Tests/de.tum.in.www1.artemis.web.rest/DataExportResource.html) | 98% | ✅|
| [RequestDataExportDTO.java](https://bamboo.ase.in.tum.de/browse/ARTEMIS-TESTS5135/latest/artifact/shared/Coverage-Report-Server-Tests/de.tum.in.www1.artemis.web.rest.dto/RequestDataExportDTO.html) | 100% | ✅|
